### DISCUSSION TITLE
Add snaps to an already existing tiddler

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#89] [Q&A] [answered]
I want to add multiple snaps to an already existing tiddler
I created these rules.

![image](https://user-images.githubusercontent.com/67494083/158078499-7268a7c6-d9ad-423b-8e74-583ddae1e359.png)

![image](https://user-images.githubusercontent.com/67494083/158078536-a470d61f-4c27-48b7-a5da-d73d906aed33.png)

But the images look like this

![image](https://user-images.githubusercontent.com/67494083/158078563-50744cd8-79e0-4c28-85f6-71432d38ef29.png)

**Comment by buggyj** [comment]
do you see the image in the file saved in  the media sub folder?

**Reply by arunnbabu81** [reply]
> do you see the image in the file saved in the media sub folder?

No I couldn’t find any file with the same name

**Comment by buggyj** [comment]
can you say the image inside the tiddlywiki using the normal snap?

**Reply by arunnbabu81** [reply]
I can save images using this rule

![image](https://user-images.githubusercontent.com/67494083/158191835-015c1f4b-33a8-4bd6-8b35-c5dbd8b4fc88.png)


But this one is not working

![image](https://user-images.githubusercontent.com/67494083/158191994-69f7bba6-67f2-4fc4-b81c-abc1e9497ec2.png)

**Comment by buggyj** [comment]
can you put the non working rule here so I can try it.

**Comment by buggyj** [comment, marked as answer]
do you have 'snap' in the mode column of the TiddlyClipConfig tiddler?

**Reply by arunnbabu81** [reply]
> do you have 'snap' in the mode column of the TiddlyClipConfig tiddler?
> […](#)

That’s a bad miss in my part.
